taskcluster/fxci_config_taskgraph/optimizations.py

```diff
@@ -0,0 +1,44 @@
+import os
+import subprocess
+from functools import cache
+
+from taskgraph.optimize.base import OptimizationStrategy, register_strategy
+from taskgraph.optimize.strategies import SkipUnlessChanged
+
+from fxci_config_taskgraph.util.integration import FIREFOXCI_ROOT_URL
```

Comment:
❗  This will conflict with #223

Reply:
Thanks for the heads up. I'm happy to rebase after that lands.